internal/controller/postgrescluster/pgadmin.go

```diff
@@ -185,53 +185,58 @@ func (r *Reconciler) reconcilePGAdminDeployment(
 
 	// if the shutdown flag is set, set pgAdmin replicas to 0
 	if cluster.Spec.Shutdown != nil && *cluster.Spec.Shutdown {
-		deploy.Spec.Replicas = initialize.Int32(0)
+		sts.Spec.Replicas = initialize.Int32(0)
 	} else {
-		deploy.Spec.Replicas = cluster.Spec.UserInterface.PGAdmin.Replicas
+		sts.Spec.Replicas = cluster.Spec.UserInterface.PGAdmin.Replicas
 	}
 
 	// Don't clutter the namespace with extra ReplicaSets.
```

Comment:
⛏️ StatefulSet doesn't have ReplicaSets.

```suggestion
	// Don't clutter the namespace with extra ControllerRevisions.
```

Reply:
Comment updated per suggestion.